src/health.ts

```diff
@@ -0,0 +1,56 @@
+import fetch from 'node-fetch';
+import { AbortSignal } from 'node-fetch/externals';
+import { URL } from 'url';
```

Comment:
I missed this the first time around, but [`URL`](https://nodejs.org/api/globals.html#url) is a global that doesn't need to be explicitly imported.